static/js/components/dashboard/util.js

```diff
@@ -16,3 +17,26 @@ export const courseListToolTip = (text: string, id: string) => (
     </ReactTooltip>
   </div>
 );
+
+export const isCurrentlyEnrollable = (enrollmentStartDate, now) => (
+  enrollmentStartDate !== null &&
+    enrollmentStartDate !== undefined &&
+    enrollmentStartDate.isSameOrBefore(now || moment(), 'day')
+);
+
+export const formatGrade = (grade) => {
+  if (_.isNil(grade) || grade === '') {
+    return '';
+  } else {
+    let gradeString;
+    if (_.isString(grade)) {
+      gradeString = _.round(Number(grade));
+    } else if (_.isInteger(grade)) {
+      gradeString = grade;
```

Comment:
I think this is unnecessary because there aren't integers in javascript, at least not in the standard types. Also, the `gradeString = _.round(Number(grade))` case should cover numbers and strings, and you've already tested against nil and an empty string, so I don't think you need the if block here at all